admin/proc-rhdh-deploy-aks.adoc

```diff
@@ -0,0 +1,315 @@
+====
+you can retrieve available regions using `az account list-locations -o table`.
+====
+
+** Create an AKS cluster:
++
+[source]
+----
+az aks create \
+--resource-group <your_ResourceGroup> \
+--name <your_ClusterName> \
+--enable-managed-identity \
+--generate-ssh-keys
+----
++
+You can refer to `--help` for additional options.
+
+** Connect to your cluster:
++
+[source]
+----
+az aks get-credentials --resource-group <your_ResourceGroup> --name <your_ClusterName>
+----
++
+The previous command configures the Kubernetes client and sets the current context in the `kubeconfig` to point to your AKS cluster.
+--
+
+* You have installed `kubectl`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html[Installing or updating kubectl].
+* You have installed Helm 3 or the latest.
+
+
+Comparison of AKS specifics with the base {product-short} deployment::
++
+--
+* *Permissions issue*: {product-short} containers might encounter permission-related errors, such as `Permission denied` when attempting certain operations. This error can be addresssed by adjusting the `fsGroup` in the `PodSpec.securityContext`.
+
+* *Ingress configuration*: In AKS, configuring ingress is essential for accessing the installed {product-short} instance. Accessing the {product-short} instance requires enabling the Routing add-on, an NGINX-based Ingress Controller, using the following commands:
++
+[source]
+----
+az aks approuting enable --resource-group <your_ResourceGroup> --name <your_ClusterName>
+----
++
+[TIP]
+====
+You might need to install the Azure CLI extension `aks-preview`. If the extension is not installed automatically, you might need to install it manually using the following command:
+
+[source]
+----
+az extension add --upgrade -n aks-preview --allow-preview true
+----
+====
+
+[NOTE]
+====
+Once you install the Ingress Controller, the 'app-routing-system' namespace with the Ingress Controller will be deployed in your cluster. Note the address of your {product-short} application from the installed Ingress Controller (for example, 108.141.70.228) for later access to the {product-short} application, later referenced as `<app_address>`.
+====
++
+[source,bash]
+----
+kubectl get svc nginx --namespace app-routing-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+----
+
+* *Namespace management*: You can create a dedicated namespace for {product-short} deployment in AKS using the following command:
++
+[source,bash]
+----
+kubectl create namespace <your_namespace>
+----
+--
+
+== Deploying the {product} on Azure Kubernetes Services (AKS) using Helm deployment
+
+You can use the following procedure to deploy your {product-short} on AKS using the Helm deployment.
+
+.Procedure
+
+. Open terminal and run the following command to add the Helm Chart repository:
++
+--
+[source]
+----
+helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+--
+
+. To create `ImagePull Secret`, run the following command:
```

Comment:
Let's maintain an active voice rather than passive, and ensure we keep the instructions clear and direct. In this case, the ImagePull secret is created using a command. After creating it, the user should proceed to edit the values.yaml file. We're presenting these steps sequentially for clarity.